sw/device/benchmarks/coremark/meson.build

```diff
@@ -38,7 +38,8 @@ foreach device_name, device_lib : sw_lib_arch_core_devices
 
   coremark_top_earlgrey_embedded = custom_target(
     'coremark_top_earlgrey_' + device_name,
-    command: make_embedded_target,
+    command: make_embedded_target_command,
+    depend_files: [make_embedded_target_depend_files,],
```

Comment:
Is the trailing comma necessary?

Reply:
It's not necessary, but our prevailing style for meson is "include the trailing comma", so that's why it's there (and yes, I favour a simpler rule rather than carving out lots of exceptions for typing a single character).